package/src/constructs/browser-check.ts

```diff
@@ -35,8 +35,17 @@ export class BrowserCheck extends Check {
       this.script = script
     } else if ('entrypoint' in props.code) {
       const entrypoint = props.code.entrypoint
+      let absoluteEntrypoint = null
+      if (path.isAbsolute(entrypoint)) {
+        absoluteEntrypoint = entrypoint
+      } else {
+        if (!this.__checkFilePath) {
+          throw new Error('You cant use relative paths without the __checkFilePath')
+        }
+        absoluteEntrypoint = path.join(this.__checkFilePath, entrypoint)
```

Comment:
I think that the test failure might be coming from here? Since __checkFilePath is a file, it might make sense to do:

`path.join(path.dirname(this.__checkFilePath), entrypoing)`